lerna.json

```diff
@@ -3,8 +3,7 @@
   "command": {
     "publish": {
       "ignoreChanges": [
-        "packages/package-json-lint/**",
-        "packages/react-config-terra/**"
```

Comment:
![image](https://user-images.githubusercontent.com/6720991/112172320-9d113900-8bc2-11eb-8a1a-5de5eb115973.png)

Reply:
Lerna still picks up changes in a 'private' repo then we get errors on publish where it won't let us push a private repo to public npm... Adding this for react-config-terra was overkill but i was being safe.

Reply:
What is `react-config-terra`?

Reply:
i think you'd call that a typo, lol